src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -105,14 +105,19 @@ TxQ::FeeMetrics::update(Application& app,
     {
         // Ledgers are taking to long to process,
         // so clamp down on limits.
-        txnsExpected_ = boost::algorithm::clamp(feeLevels.size(),
-            minimumTxnCount_, targetTxnCount_);
+        auto const cutPct = 100 - setup.slowConsensusDecreasePercent;
```

Comment:
do we need to guard against `slowConsensusDecreasePercent` being > 100 somewhere? Even if we're ok with the underflow in terms of the clamp operation, it's probably not what the user intended.

Reply:
I had thought about 100% leading the result to be 0, which will be clamped back up to `minimumTxnCount_`, but your question made me realize that > 100 will overflow and result in the limit being crazy large, which would be bad. I've added some range checking to `setup_TxQ`.